src/components/RequestButton/index.tsx

```diff
@@ -550,7 +550,6 @@ const RequestButton: React.FC<RequestButtonProps> = ({
                 onClick={button.action}
                 key={`request-option-${button.id}`}
               >
-                {button.svg}
```

Comment:
Why is this removed?

Reply:
As noted above:

> Also made one small/minor change to remove the icon from the regular "request" button dropdown children on movie/series detail pages, since there are no icons in other dropdowns (e.g., the "play" button dropdown children) and we should be consistent. (If we want to add the icons to the request dropdowns, then we should add them everywhere else too!)

Reply:
I'm totally cool with putting it back, but then I feel we should update the "play" button dropdown children to have icons as well.  Lmk what you think!

Reply:
I disagree with this change. Consistency doesn't directly always translate to clarity. The request dropdown has a lot of mixed options that do different things. The icons help quickly understand what you are clicking on at a glance.

Reply:
All of the actions for the `PlayButton` are "watching" something. They would all have the same icon. It would look silly.

Reply:
Hmmm okay.  I'll put it back 😄